app/src/main/res/layout/fragment_my_books.xml

```diff
@@ -25,7 +25,6 @@
         android:layout_marginRight="24dp"
         android:layout_marginBottom="16dp"
         android:contentDescription="@string/fab_content_desc"
-        android:onClick="addBook"
```

Comment:
This was unused and was causing the Travis CI build to fail. Weird how our build ignored this error